chapters/data-collection.tex

```diff
@@ -3,57 +3,58 @@
 \begin{fullwidth}
 Much of the recent push toward credibility in the social sciences has focused on analytical practices.
 However, credible development research often depends, first and foremost, on the quality of the raw data.
-This is because, when you are collecting the data yourself,
-or it is provided only to you through a unique partnership,
-there is no way for others to validate that it accurately reflects the reality
+When you are using original data - whether collected for the first time through surveys or sensors or acquired through a unique partnership -
+there is no way for others to validate that it accurately reflects reality
```

Comment:
There is something about this sentence that rubs me a little the wrong way. I think it was fine to say "no way for others" when we talked about data collection only, but but when we include admin data it does not feel as right, as the civil servants that gave us the data could validate it. How about this suggestion:

```suggestion
there is no way for anyone else in the research community to validate that it accurately reflects reality
```